integration-tests/actions/vrfv3_helpers.go

```diff
@@ -0,0 +1,352 @@
+package actions
```

Comment:
This file should probably be renamed to ocrv2vrf_helpers.go since the v3 naming is being changed everywhere else.

Reply:
+1. applies to other file names that have `vrfv3`

Reply:
renamed